### ISSUE TITLE
Multiple spaces are stripped when using text API

### ISSUE THREAD

**Issue by jjcohen** [PiSupply/PaPiRus#210]
Hi I recently set up a Papirus and Raspberry Pi 3 (not used for anything else, fresh install etc), both of which I bought a while back.
Loving the display it's great however it seems that multiple spaces are swallowed by the PapirusTextPos. For instance "hello     word" would be printed as "hello world".

- I did the manual install as the automatic install did not work for me.
- This is the version of the git repo I have locally : c6da9c123081a620760865460255b65d2c1d9adc
- The following is the script I ran:
`from papirus import PapirusTextPos

text = PapirusTextPos()

text.AddText("hello                            world", 10, 10, Id="Start" )`

-  Here is what is displayed on the physical device:
![20191023_084528](https://user-images.githubusercontent.com/1882987/67401615-4f486980-f5a7-11e9-8dd1-10e5b92a0c50.jpg)

**Comment by tvoverbeek** [comment]
Yes, the text is split into words to calculate how many words of the text fit on a single line before
starting a new line.
See the code of the addToImageText function around line 112 in textpos.py (https://github.com/PiSupply/PaPiRus/blob/master/papirus/textpos.py).
If you really need that much space between words, use two separate calls to AddText with the same y but different x coordinates.
